assets/js/src/common/categories/categories.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+
+type Category = {
+  name: string,
+  label: string,
+  count?: number | string,
+};
+
+type Props = {
+  onSelect: (category: any) => any,
```

Comment:
I see you are calling the `onSelect` with the `category.name` so its argument is `name: string`, right?